package.json

```diff
@@ -42,18 +42,20 @@
     "files": [
         "dist/**/*"
     ],
+    "sideEffects": false,
```

Comment:
a field that informs Webpack that this package has no files which produce side effects when imported